1. Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

Используем формулу равномерного распределения на отрезке 

                                (а + b)
Математическое ожидание  М(X)  ---------
                                   2 
Среднее значение равно: 500

In [10]:
mx = (200 + 800) / 2
mx

500.0

               (b - a)**2
Дисперсия D(X) ---------
                  12 
Дисперсия: 30000

In [11]:
dx = (800 - 200) ** 2 / 12
dx

30000.0

2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.


Найдем правую границу b величины B  
D(B) = 0.2
a = 0.5

In [12]:
b=0.5+2.4**(1/2)
b

2.049193338482967

И найдем среднее значение по формуле:
        a + b
M(B )= -------
         2

In [13]:
avg = (b+0.5)/2
avg

1.2745966692414834

3. Непрерывная случайная величина X распределена нормально и задана плотностью 
распределения f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32)
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)


Задачу не понял(

4. Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

Для расчета значений вероятности воспользуемся, методом cdf, объекта NormalDist стандартного модуля statistics

In [15]:
from statistics import NormalDist

Напишем функцию для определения Z значений для данного распределения:

In [16]:
def z_value(height):
    return (height-174)/8

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см

In [17]:
print(z_value(182))

1.0


метод cdf определяет вероятность того, что случайная величина X будет меньше или равна x, т.е в данном случае P(X<=182)

In [18]:
NormalDist().cdf(z_value(182))

0.8413447460685429

Так как нам нужна вероятность P(X<=182), поэтому вычтем полученное значение из 1:

In [19]:
P=1-NormalDist().cdf(z_value(182))
P

0.15865525393145707

Ответ: P(X<=182) = 0.1586

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
б). больше 190 см

In [20]:
z=z_value(190)
z

2.0

Так как нам нужна вероятность P(X<=190), поэтому вычтем полученное значение из 1:

In [21]:
P=1-NormalDist().cdf(z)
P

0.02275013194817921

Ответ: P(X<=190) = 0.0228

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
в). от 166 см до 190 см

In [22]:
z1=z_value(166)
z1

-1.0

In [23]:
z2=z_value(190)
z2

2.0

P(166 <= X <= 190 )

In [24]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.8185946141203637

Ответ: P(166 <= X <= 190 ) = 0.8186

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
г). от 166 см до 182 см

In [25]:
z1=z_value(166)
z1

-1.0

In [26]:
z2=z_value(182)
z2

1.0

P(166 <= X <= 182 )

In [27]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.6826894921370859

Ответ: P(166 <= X <= 182 ) = 0.6827

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
д). от 158 см до 190 см

In [28]:
z1=z_value(158)
z1

-2.0

In [29]:
z2=z_value(190)
z2

2.0

P(158 <= X <= 190 )

In [30]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
P

0.9544997361036416

Ответ: P(158 <= X <= 190 ) = 0.9545

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
е). не выше 150 см или не ниже 190 см

In [31]:
z1=z_value(150)
z1

-3.0

In [32]:
z2=z_value(190)
z2

2.0

P(X <= 150 или Х >= 190)

In [33]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.024100029979809312

Ответ: P(X <= 150 или Х >= 190) = 0.0241

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
ё). не выше 150 см или не ниже 198 см

In [34]:
z1=z_value(150)
z1

-3.0

In [35]:
z2=z_value(198)
z2

3.0

P(X <= 150 или Х >= 198)

In [36]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
P

0.002699796063260207

Ответ: P(X <= 150 или Х >= 198) = 0.0027

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
ж). ниже 166 см.

In [37]:
z=z_value(166)
z

-1.0

P(X <= 166) 

In [38]:
P=NormalDist().cdf(z)
P

0.15865525393145707

Ответ: P(X <= 166) = 0.1587

5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

Воспользуемся формулой расчета Z оценки

In [39]:
Z=(190-178)/25**(1/2)
Z

2.4

Ответ: на 2.4 сигмы